lago/vm.py

```diff
@@ -266,9 +266,7 @@ def extract_paths(self, paths, ignore_nopath):
 
     @_check_defined
     def vnc_port(self):
```

Comment:
As this drops entirely VNC support, I think its better to leave no trace of it(remove this function and others that relay on it)